### ISSUE TITLE
Drop the accessibility warning for downloadable slides with annotation

### ISSUE THREAD

**Issue by antobinary** [bigbluebutton/bigbluebutton#19921]
**Describe the bug**
With the improvements made in https://github.com/bigbluebutton/bigbluebutton/pull/19082 we no longer need to warn users about lacking accessibility on the downloadable slides+annotations.  (@KDSBrowne could you please re-confirm?)

**To Reproduce**
1. Export slide with annotation
2. hover over the (i) in the public chat

**Screenshots**
![image](https://github.com/bigbluebutton/bigbluebutton/assets/6312397/90280cc8-596f-4450-954c-6a4b7ed37f18)

**BBB version:**
BigBlueButton 3.0.0-alpha.6

**Comment by KDSBrowne** [comment]
@antobinary,  We can remove accessibility warning now that the text is selectable.
